README.md

```diff
@@ -1,8 +1,50 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Marissa Schmitt . Jarvis Consulting
 
+I am a recent graduate from Conestoga College, where I studied Software Engineering Technology. I am excited to be challenged by new technologies and expand my skill set as I have learned various programming languages during the duration of my program. During my time at Conestoga College, I have gained experience in web development, mobile development, RDBMS, OOP, and assembly language. My interest in software engineering started in high school, where I pursued many computer science and computer engineering courses. I am intrigued by all aspects of software engineering, whether front-end, back-end, or directly working with a database. Currently, I am working as a software developer and data engineer at Jarvis to improve my skills by learning the latest technologies.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C++, C#, C, JavaScript (Vue and ReactJS), MongoDB
+
+**Familiar:** .NET, Swift, Kotlin, Assembly language, Waterfall Methodology
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent to manage and insert resource data, such as CPU, RAM, disk usage into a PostgreSQL database hosted by Docker. The host usage information was collected through bash commands and automated using crontab. The data collected by the app can be returned as a result set by using SQL queries which also includes detecting crontab insertion failure using SQL aggregate functions to compare the number of entries within a span of five minutes. Other information, such as retrieving number of CPUs and amount of total memory can be used for data analytics when query the database. This app was developed and run within a CentOS7 environment.
```

Comment:
Implemented a Linux cluster monitoring agent to manage and insert resource data, such as CPU, RAM, disk usage into a PostgreSQL database hosted by Docker. The host usage information was collected through bash commands and automated using crontab. The data collected by the app can be returned as a result set by using SQL queries, including detecting crontab insertion failure using SQL aggregate functions to compare the number of entries within a span of five minutes. Other information, such as retrieving the number of CPUs and the amount of total memory can be used for data analytics when querying the database. This app was developed and run within a CentOS 7 environment.